scripts/generate-ledger-conformance-test-cases.sh

```
@@ -0,0 +1,57 @@
+        # Iterate through files in network directory
+        for FILE_PATH in "$TEST_GROUP"/*; do
+            if [[ -f "$FILE_PATH" ]]; then
+                FILE_NAME=$(basename "$FILE_PATH")
+                TEST_CASES+=("$FILE_PATH")
+            fi
+        done
+    fi
+done
+
+# Sort test cases
+IFS=$'\n' TEST_CASES=($(sort <<<"${TEST_CASES[*]}"))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)